pkg/state/conformance/state.go

```diff
@@ -680,7 +685,12 @@ func (suite *StateSuite) TestWatch() {
 	// get event history
 	chWithTail := make(chan state.Event)
 
-	suite.Require().NoError(suite.State.Watch(ctx, path1.Metadata(), chWithTail, state.WithTailEvents(1000)))
+	err := suite.State.Watch(ctx, path1.Metadata(), chWithTail, state.WithTailEvents(1000))
+	if state.IsUnsupportedError(err) {
+		return
```

Comment:
so I guess we don't run other tests if the backend doesn;t support watching?

Reply:
Watch is supported, only `tailEvents` (history replay) isn't. It is easier to review this hand in hand with the other PR: https://github.com/cosi-project/state-etcd/pull/1/files#diff-f4f4e768803041fd484adb327aebc9e2d23af18dbf33b854a0489207032c3940R281-R283